index.html

```diff
@@ -8,20 +8,19 @@
     <title>Faranosh Portfolio</title>
   </head>
   <body>
-    <header class="toolbar">
-      <div class="logo">
-        <a id="LOGO" href="#"><b>My Logo</b></a>
-      </div>
-      <nav class="icon">
-        <img src="Images/Mobile-version/Union.png" alt="Menu icon" />
-      </nav>
-      <div class="menu">
-        <ul class="menu-bar">
-          <li class="MB"><a href="#">Portfolio</a></li>
-          <li class="MB"><a href="#">About</a></li>
-          <li class="MB"><a href="#">Contact</a></li>
+    <div class="home-page">
+    <header>
+      <nav id="nav">
+        <a class="logo" href="#"><img src="Images/logo.png" alt="my logo"/></a> 
+  
+        <nav class="hamburger"><img src="Images/Mobile-version/Union.png" alt="Menu"></nav>
+        <ul class="mainmenu">
+          <li><a href="#"></a></li>
+           <li><a href="#work">Portfolio</a></li>
+           <li><a href="#about_me">About</a></li>
+           <li><a href="#contact"> Contact </a></li>
```

Comment:
- [ **OPTIONAL** ] I think it would be better to improve the styling of the navigations list in the Desktop version Like:
   - Improving the spacing between navigations list elements (portfolio, About, contact)
   - Removing list-styling on all the options(on Desktop): you can do this by only adding `list-style: none;` in the code below in your CSS file instead of repeating yourself every time.
    ```bash
    * {
        box-sizing: border-box;
        margin: 0;
        padding: 0;
        font-family: "poppins", sans-serif;
         ....
    }
   ```
- [ **OPTIONAL** ] I also suggest you improve the **font-weight** on the navigations list(portfolio, About, contact) on the mobile menu to make it look smart. You can check the used value on Figma template.

Reply:
Thanks a lot for your useful feedback.